webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -837,6 +802,12 @@ const AdvancedTextEditor = ({
                 noArgumentHandleSubmit={handleSubmitWrapper}
                 isInEditMode={isInEditMode}
             />
+            {isInEditMode && (
```

Comment:
Duplicate change here from the other PR

Reply:
it should be since when the other gets merged